.eslintrc.cjs

```diff
@@ -14,5 +15,14 @@ module.exports = {
       'warn',
       { allowConstantExport: true },
     ],
+    'no-console': 'off',
```

Comment:
Not the best idea :)
Yoy need to prevent unexpected consoles in the comitted code